### ISSUE TITLE
Native <select> loses focus after closing it with ESC key for JAWS + Chrome/Edge

### ISSUE THREAD

**Issue by Karkhutvy** [FreedomScientific/VFO-standards-support#901]
**Summary**
Pressing ESC key for opened native HTML select with JAWS active causes focus to be moved to browser(Chrome or Edge) settings menu instead of being returned to 'select' element.
Reproduced for basic examples like this:
```
<label for="fruit">Choose a fruit:</label>
<select id="fruit" name="fruit">
  <option value="">--Please choose an option--</option>
  <option value="apple">Apple</option>
  <option value="banana">Banana</option>
  <option value="orange">Orange</option>
  <option value="grape">Grape</option>
</select>
```
https://jsfiddle.net/fv9p7htq/

 **Steps to reproduce:**

1 Start JAWS
2 Start Chrome/Edge
3 Open any HTML select to see dropdown options(For example this base example - https://jsfiddle.net/fv9p7htq/)
4 Press ESC to close dropdown

**Actual result:**
Dropdown is closed but focus is set on browser 'Settings' button
https://github.com/user-attachments/assets/c62c6051-06a5-49c1-99b4-2726b8bcc217

**Expected result:**
Dropdown is closed but focus should be set on 'select' element as it works without JAWS active

Issue is not reproduced for JAWS + Firefox or NVDA + Chrome
https://github.com/user-attachments/assets/a177b0f4-1de0-4183-85d9-423fffac0798

**Additional Information:**

JAWS 2025.2504.89
Chrome Version 136.0.7103.49 (Official Build) (64-bit) or Edge Version 136.0.3240.50 (Official build) (64-bit)
Windows 11 Enterprise 26100.3775

**Comment by BrettLewisVispero** [comment]
This should be fixed in the next jaws 2025 release.
